src/uk/gov/hmcts/pipeline/TeamConfig.groovy

```diff
@@ -0,0 +1,94 @@
+package uk.gov.hmcts.pipeline
+
+import org.apache.commons.lang.StringUtils
+
+class TeamConfig {
+
+  def steps
+  static final String GITHUB_CREDENTIAL = 'jenkins-github-hmcts-api-token'
+  static final String DEFAULT_TEAM_NAME = 'pleaseTagMe'
+  static final String NAMESPACE_KEY = "namespace"
+  static final String CONTACT_SLACK_CHANNEL_KEY = "contact_channel"
+  static final String BUILD_NOTICES_CHANNEL_KEY = "build_notices_channel"
+  static final String TEAM_KEY = "team"
+  static final String SLACK_KEY ="slack"
+  static def teamConfigMap
+  def appPipelineConfig
+
+  TeamConfig(steps){
+    this.steps = steps
+  }
+
+  TeamConfig(steps, appPipelineConfig){
```

Comment:
do you think it would be crazy to pass in the team name here as well, that way the methods could change to plain getters?
```
.rawProductName
.buildNoticesSlackChannel
```
etc

Reply:
Simplifies in this case, but don't like constructor having it.